src/App.svelte

```diff
@@ -1,372 +1,26 @@
-									<button on:click={completeTask(task)}><Emoji symbol="🏆" label="Completed"></Emoji></button>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to edit this task.">
-									<td contenteditable="true" bind:innerHTML={task.text}/>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to delete this task from the list.">
-									<button on:click={() => deleteTask(task)}><Emoji symbol="🗑️" label="Delete"></Emoji></button>
-								</ToolTip>
-							</td>
-						</tr>
-					{/each}
-				{:else if mode === "active"}
-					{#each activeTasks as task}
-						<tr class:done={task.done}>
-							<td>
-								<ToolTip title="Click to toggle whether this task has been completed.">
-									<button on:click={completeTask(task)}><Emoji symbol="🏆" label="Completed"></Emoji></button>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to edit this task.">
-									<td contenteditable="true" bind:innerHTML={task.text}/>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to delete this task from the list.">
-									<button on:click={() => deleteTask(task)}><Emoji symbol="🗑️" label="Delete"></Emoji></button>
-								</ToolTip>
-							</td>
-						</tr>
-					{/each}
-				{:else}
-					{#each completedTasks as task}
-						<tr class:done={task.done}>
-							<td>
-								<ToolTip title="Click to toggle whether this task has been completed.">
-									<button on:click={completeTask(task)}><Emoji symbol="🏆" label="Completed"></Emoji></button>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to edit this task.">
-									<td contenteditable="true" bind:innerHTML={task.text}/>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to delete this task from the list.">
-									<button on:click={() => deleteTask(task)}><Emoji symbol="🗑️" label="Delete"></Emoji></button>
-								</ToolTip>
-							</td>
-						</tr>
-					{/each}
-				{/if}
-			</table>
-		</div>
-
-		<footer>
-			<div class="counter">
-				{#if count === 1}
-					{count} task remaining
-				{:else}
-					{count} tasks remaining
-				{/if}
-			</div>
-			<div class="spacer">|</div>
-			<div class="mode">
-				Showing {mode} tasks
-			</div>
-			<div class="spacer">|</div>
-			<div class="filterAll">
-				<button on:click={() => filterAllTasks()}>All</button>
-			</div>
-			<div class="spacer">|</div>
-			<div class="filterActive">
-				<button on:click={() => filterActiveTasks()}>Active</button>
-			</div>
-			<div class="spacer">|</div>
-			<div class="filterCompleted">
-				<button on:click={() => filterCompletedTasks()}>Completed</button>
-			</div>
-			<div class="spacer">|</div>
-			<div class="clear">
-				<button on:click={() => clearAllTasks()}>Clear All Tasks</button>
-			</div>
-			<div class="spacer">|</div>
-			<div class="toggle">
-				<button on:click={() => toggleAlerts()}>Toggle Alerts</button>
-			</div>
```

Comment:
You don't need to wrap each `<button>` element in a `<div>` and you don't need to have a different `class` for each one (since they all had the same CSS rule). I moved these to the `<Footer>` component and used [Flexbox](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox) to space out the elements.